src/Core/Configure.php

```diff
@@ -252,23 +251,32 @@ public static function consume($var)
      * @param \Cake\Core\Configure\ConfigEngineInterface $engine The engine to append.
      * @return void
      */
-    public static function config($name, ConfigEngineInterface $engine)
+    public static function config(string $name, ConfigEngineInterface $engine): void
     {
         static::$_engines[$name] = $engine;
     }
 
+    /**
+     * Returns true if the Engine objects is configured.
+     *
+     * @param string $name Engine name.
+     * @return bool
+     */
+    public static function isConfigured(string $name): bool
```

Comment:
I think this is a reasonable change. We could add a deprecation warning/backport this method to 3.next as well so anyone using the old signature can upgrade early.